experimental/datacommons-js/example.html

```diff
@@ -52,6 +52,20 @@ <h2>Map</h2>
       enclosedPlaceType="County"
       statVarDcid="Annual_ExpectedLoss_NaturalHazardImpact"
     ></datacommons-map>
+    <h2>Map with year slider</h2>
+    <datacommons-map
+      title="Population"
+      place="country/USA"
+      childPlaceType="State"
+      subscribe="dc-map"
+      variable="Count_Person"
+    ></datacommons-map>
+    <datacommons-slider
+      max="2023"
+      min="1950"
+      publish="dc-map"
+      value="2023"
+    ></datacommons-slider>
```

Comment:
I think "publish" and "subscribe" are not intuitive for people with limited coding backgrounds. Perhaps we can rename them to something like:
```
<datacommons-slider
...
name="time-slider-1"
></datacommons-slider>

<datacommons-map>
...
controlledBy="time-slider-1"
></datacommons-map>
```
@dwnoble @shifucun @pradh thoughts?

Reply:
That is fair about using the names `publish` / `subscribe`. I don't think `name`/`controlledBy` is optimal either though- there could be a many-to-many relationship between visualization components and controls. for example, a `datacommons-map` could be controlled by both a time slider and a to-be-created country picker. In this case the time slider and country picker would both broadcast on the same channel.

Alternative possible names
`publish`: `publishEvent`,   `triggerEvent`, `emit`, `fire`, `broadcast`, `channel`,
`subscribe`: `listen`, `on`, `handle`, `channel`

Another thought: Users can trigger these events directly from their own JS code and have a datacommons web component listen for them. We should balance making it clear what the functionality is for JS-savvy users, and that these attributes are JS event names for more advanced users.

Reply:
Sounds good. Happy with going with publish/subscribe for now, we can update if we think there's a better naming convention.